mppm/install.md

```diff
@@ -11,18 +11,21 @@ To begin testing your multiplayer functionality, you must first install Multipla
 To install Multiplayer Play Mode, you must have:
 
 - Unity Editor version 2023.1 or later
-- An active Project in your Unity Hub
+- An active Project in Unity Hub
 
 
 ## Install the Multiplayer Play Mode package
 
-1. Open your Unity Hub and select the Project your are presently working on.
-2. In the editor's menu bar, navigate to **Window** > **Package Manager**.
-3. Click the plus sign ![plus sign](/img/add.png) in the Package Manager status bar and select **Add package by name...**.
-4. Enter `com.unity.multiplayer.playmode` in the pop-up window and select **Add**.
-5. Once the download is complete, **Multiplayer Play Mode** appears under **Unity Technologies** in your **Package Manager** window.
-6. To open the Multiplayer Play Mode window, navigate to **Window** > **Multiplayer Play Mode**.
-
-## Next Steps
+1. Open Unity Hub.
+2. Select the Project you are working on.
```

Comment:
I'm inclined to change this to "Select a Project". What do you think?

Reply:
Agree - thanks :)